ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.10__amd64-stable.yaml

```diff
@@ -53,13 +53,14 @@ tests:
     test:
     - chain: cucushift-installer-check-cluster-health
     workflow: cucushift-installer-rehearse-aws-c2s-ipi-disconnected-private
-- as: aws-ipi-disconnected-cco-manual-sts-ep-p2-f28
-  cron: 3 8 23 * *
+- as: aws-ipi-disconnected-cco-man-sts-ep-fips-p2-f28
```

Comment:
We had ever talked with sts naming issue before, cc @gpei and @yunjiang29 , we suggest to remove "cco-manual" keyword directly for any sts ci jobs.

Reply:
yes, we had a discussion with CCO team and reached the following agreement:

- use `sts` to replace `cco-manual-sts` for aws
- use `workload-identity` to replace `cco-manual-workload-identity` for gcp

Reply:
@jianlinliu @gpei  Replaced `cco-manual-sts` and `cco-man-sts` with `sts`.
I think we can do the other rename in a follow up PR, as this PR already contains many lines of changes.

Reply:
Applied the changes for both `cco-manual-sts` and `cco-manual-workload-identity`